corehq/apps/hqmedia/README.md

```diff
@@ -46,7 +46,7 @@ CommCare (on mobile) uses the version to determine when to re-download an update
 
 The version is updated (set to the latest app version) in the following situations:
 - Multimedia is added to the app.
-- The content of multimedia is changed, by using the "Replace" button in the uploader. This also causes the multimedia id to change. It typically does not change the unique_id, because `set_media_versions` looks up the path in the previous version of the app and uses that item's unique id.
+- The content of multimedia is changed, by using the "Replace" button in the uploader. This also causes the multimedia id to change. It typically does not change the unique_id, because `set_media_versions` looks up the path in the previous version of the app and uses that item's unique id. It may change the unique_id for form question media, even if the file was in the previous version, because form questions include a random string at the end of their path, and this string will change when a user uploads a new file.
```

Comment:
do you mean "path" for form question media? the "unique_id" in the second statement makes me think these two are the same models.

Reply:
I do mean `unique_id`. Would it make more sense to say "For most places that an app can reference media, this does not change the unique_id... But for form question media, it may change the unique_id because..."?

Reply:
okay, i understand this now.. took some reads :)

i think this will be better as
`Although, It may change the unique_id for form question's media, because form questions include a random string at the end of their path which changes when a user uploads a new file`

Reply:
https://github.com/dimagi/commcare-hq/pull/25341 - thank you for the thoughtful feedback.